src/library/scala/util/control/Exception.scala

```diff
@@ -46,10 +46,11 @@ object Exception {
     def apply(x: Throwable): T = f(downcast(x).get)
   }
 
-  def mkThrowableCatcher[T](isDef: Throwable => Boolean, f: Throwable => T) = mkCatcher(isDef, f)
+  def mkThrowableCatcher[T](isDef: Throwable => Boolean, f: Throwable => T): Catcher[T] =
```

Comment:
Not needed anymore.  Adding explicit type parameters to `mkCatcher(isDef, f)` is enough now.